code/game/turfs/open/space/space.dm

```diff
@@ -35,6 +75,10 @@ GLOBAL_VAR_INIT(starlight_color, COLOR_STARLIGHT)
 	//This is used to optimize the map loader
 	return
 
+/turf/open/space/Destroy()
+	. = ..()
+	GLOB.starlight -= src
```

Comment:
```suggestion
	GLOB.starlight -= src
	return ..()
```

i mean there's likely zero difference either way but i always feel awk about invoking parent _then_ cleaning ourselves up. you can ignore this if you think it's un-necessary